activemodel/lib/active_model/naming.rb

```diff
@@ -195,28 +195,38 @@ def initialize(klass, namespace = nil, name = nil, locale = :en)
     #
     # Specify +options+ with additional translating options.
     def human(options = {})
-      return @human unless @klass.respond_to?(:lookup_ancestors) &&
-                           @klass.respond_to?(:i18n_scope)
-
-      defaults = @klass.lookup_ancestors.map do |klass|
-        klass.model_name.i18n_key
-      end
+      return @human unless i18n_keys.present?
```

Comment:
This line is not the same condition as before.

```suggestion
      return @human if i18n_keys.empty? || i18n_scope.empty?
```

Reply:
Yes, I based this simplification on [`ActiveModel::Translation`](https://github.com/rails/rails/blob/ed8e7672180aff0101da0826087bf4ca07094f02/activemodel/lib/active_model/translation.rb) implementing both `lookup_ancestors` and `i18n_scope`.

It is possible that `@klass` implements `lookup_ancestors` independently.  My opinion was that would be an extreme corner case in the context of `ActiveModel::Name`.

But, as you suggest, I will preserve the previous condition exactly. :+1: